scripts/hook_injector.py

```diff
@@ -0,0 +1,100 @@
+#!/usr/bin/env python3
+'''
+hook_injector.py
+
+    This example harnesses the power of
+    Manticore as a debugger during program analysis,
+    providing introspection of hooks injected through
+    user-specified pattern inputs.
```

Comment:
Any chance to have a minimal working example here? E.g. a little C code and its output example? Thanks to that the script usability would be immediately visible for future readers :D.